hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -388,6 +388,7 @@ forAllFanout action =
        in action utxo tx
             & label ("Fanout size: " <> prettyLength (countAssets $ txOuts' tx))
  where
+  -- TODO: Can we go back to 70 outputs here?
   maxSupported = 70
```

Comment:
how has removing the traces solved this TODO?

Reply:
Removing traces reduced tx sizes so I could revert to the old number of outputs we had.

Reply:
I removed the line also